cmd/res/profiles/ED_ID1.yaml

```diff
@@ -0,0 +1,114 @@
+name: "ED_ID1"
+manufacturer: "HCL Corp."
+model: "Some 3rd party app sending data"
+labels:
+ - "rest"
+ - "ED_ID1"
+description: "REST Device"
+
+deviceResources:
+  - name: jsonRes
+    isHidden: false
+    description: "Test resource 1 Message"
+    properties:
+      valueType: "Object"
+      readWrite: "RW"
+      mediaType: "application/json"
+  - name: stringRes
+    isHidden: false
+    description: "used to read DUT Info from the End device"
+    properties:
+        valueType: "String"
+        readWrite: "RW"
+        mediaType: "text/plain"
```

Comment:
`mediaType` only needed and used for Binary and Object types. Please remove from these resources.